.gitignore

```diff
@@ -168,3 +168,6 @@ cython_debug/
 
 # generated changelog
 /docs/changelog.md
+.zed
+.cursorrules
+.cursorignore
```

Comment:
just my latest thoughts but I feel like IDE specifics could go into a global gitignore (https://sebastiandedeyne.com/setting-up-a-global-gitignore-file) instead of per project to keep it lean.

No fuss now, though! :)